servers/text_server.h

```diff
@@ -66,8 +66,15 @@ class TextServer : public Object {
 		BREAK_NONE = 0,
 		BREAK_MANDATORY = 1 << 4,
 		BREAK_WORD_BOUND = 1 << 5,
-		BREAK_GRAPHEME_BOUND = 1 << 6
-		//RESERVED = 1 << 7
+		BREAK_GRAPHEME_BOUND = 1 << 6,
+		BREAK_WORD_BOUND_ADAPTIVE = 1 << 5 | 1 << 7
+	};
+
+	enum TextOverrunFlag {
```

Comment:
Missing `VARIANT_ENUM_CAST(TextServer::TextOverrunFlag);` at the end of file and `BIND_ENUM_CONSTANT(...)`s in the text server `_bind_methods()`.

Reply:
Fixed.